### PR TITLE
CI: no deps bundle size check

### PR DESCRIPTION
**Pull request by signed-long**
Adds a small report with the files contributing the most to the bundle size + their dependencies

https://github.com/signed-long/new-connect/actions/runs/9916882443/job/27399715452#step:6:1

### CODE REVIEW

File: src/ci/dependency_report.cjs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+report = [];
+for (const mapFile of mapFiles) {
+    var mapFileData = JSON.parse(fs.readFileSync(dir + mapFile, 'utf8'));
+    assetFile = mapFile.replace(extension, "");
+    sources = mapFileData.sources.filter(source => source.includes("node_modules")).map(source => source.split("node_modules/")[1]);
+    report.push({ "asset": assetFile, "size": fs.statSync(dir + assetFile).size, "sources": sources});
+    fs.unlinkSync(dir + mapFile);
+}
+report.sort((b, a) => a.size - b.size).forEach(entry => {
+    console.log(entry.asset + ":" + (entry.size / 1024).toFixed(2) + "KB");
```
Comment (adeebshihadeh) [inline src/ci/dependency_report.cjs:18]: two spaces

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -28,17 +28,10 @@ jobs:
       - uses: oven-sh/setup-bun@v1
 
       - run: bun install --frozen-lockfile
-      - run: bun vite-bundle-visualizer -o ./bundle-visualizer.html
+      - run: bun run build --sourcemap true
+      - run: node src/ci/dependency_report.cjs dist/ | column -t -s ":"
```
Comment (adeebshihadeh) [inline .github/workflows/build.yaml:32]: let's give this a good name

**Review by adeebshihadeh — APPROVED** [review]
Much better! Very curious why HLS.js is so big.

File: .github/workflows/build.yaml (hunk 2 above)
Comment (signed-long) [reply .github/workflows/build.yaml]: Hows `Bundle size breakdown` ?

**Pull request merged**